packages/eslint-plugin-template/tests/rules/no-any.test.ts

```diff
@@ -43,43 +43,52 @@ ruleTester.run(RULE_NAME, rule, {
     convertAnnotatedSourceToFailureCase({
       description: 'it should fail with call expression in expression binding',
       annotatedSource: `
-      {{ $any(framework).name }}
-         ~~~~~~~~~~~~~~~
+        {{ $any(framework).name }}
+           ~~~~~~~~~~~~~~~
       `,
       messageId,
+      suggestions: [
+        {
+          messageId: suggestRemoveAny,
+          output: `
+        {{ framework.name }}
+           
+      `,
+        },
+      ],
     }),
     convertAnnotatedSourceToFailureCase({
       description: 'it should fail with call expression using "this"',
       annotatedSource: `
-      {{ this.$any(framework).name }}
-         ~~~~~~~~~~~~~~~~~~~~
+        {{ this.$any(framework).name }}
+           ~~~~~~~~~~~~~~~~~~~~
       `,
       messageId,
+      suggestions: [
+        {
+          messageId: suggestRemoveAny,
+          output: `
+        {{ this.framework.name }}
+           
+      `,
+        },
+      ],
     }),
     convertAnnotatedSourceToFailureCase({
       description: 'it should fail with call expression in property binding',
       annotatedSource: `
-      <a [href]="$any(getHref())">Click here</a>
-                 ~~~~~~~~~~~~~~~
+        <a [href]="$any(getHref())">Click here</a>
+                   ~~~~~~~~~~~~~~~
       `,
       messageId,
-    }),
-    convertAnnotatedSourceToFailureCase({
-      description: 'it should fail for multiple cases',
-      annotatedSource: `
-        {{ $any(framework).name }}
-           ~~~~~~~~~~~~~~~
-       <div>
-        {{ this.$any(framework).name }}
-           ^^^^^^^^^^^^^^^^^^^^
-        </div>
-        <a [href]="$any(getHref())">Click here</a>'
-                   ###############
+      suggestions: [
+        {
+          messageId: suggestRemoveAny,
+          output: `
+        <a [href]="getHref()">Click here</a>
+                   
       `,
-      messages: [
-        { char: '~', messageId },
-        { char: '^', messageId },
-        { char: '#', messageId },
```

Comment:
I removed this because in addition to looking redundant, since we already have these tests, I was unable to make multiple `suggestions` work at the same time.

Reply:
In the past I definitely remember this particular test being useful for catching location information regressions, so I am hesitant to change it...

Have you tried seeing if you can make it work with multiple suggestions using the raw RuleTester, i.e. removing the `convertAnnotatedSourceToFailureCase` util initially?

Reply:
Oh, I forgot to leave a message here saying that I found the cause for this... the fix is in #491.